vortex-ffi/src/duckdb/mod.rs

```diff
@@ -0,0 +1,88 @@
+use std::cmp::min;
+use std::ffi::c_uint;
+
+use duckdb::core::DataChunkHandle;
+use duckdb::ffi::{duckdb_data_chunk, duckdb_logical_type};
+use vortex::compute::slice;
+use vortex::dtype::DType;
+use vortex::error::VortexExpect;
+use vortex::{Array, ToCanonical};
+use vortex_duckdb::{ToDuckDBType, to_duckdb_chunk};
+
+use crate::array::FFIArray;
+
+/// This is the default chunk size for duckdb.
+/// It is best to return data chunks of this size to duckdb.
+const DUCKDB_STANDARD_VECTOR_SIZE: usize = 2048;
```

Comment:
Can keep for now, but would be nice to get that from duckdbs c api.

Reply:
good idea

Reply:
I will follow up